src/api/subscriptions.js

```diff
@@ -116,6 +116,12 @@ export class Subscriptions {
    */
   setOnLoginRequest(callback) {}
 
+  /**
+   * Starts the LoginPrompt flow.
```

Comment:
Is the "waiting" API coming in a separate PR?

Reply:
Yes

Reply:
yes